components/admin_console/billing/billing_subscriptions/billing_subscriptions.tsx

```diff
@@ -0,0 +1,229 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React from 'react';
+import {FormattedMessage} from 'react-intl';
+
+import {trackEvent} from 'actions/telemetry_actions';
+
+import BlockableLink from 'components/admin_console/blockable_link';
+import AlertBanner from 'components/alert_banner';
+
+import privateCloudImage from 'images/private-cloud-image.svg';
+
+export const contactSalesCard = (contactSalesLink: any, isFreeTrial: boolean, isPaidTier: boolean) => {
+    const title = {id: '', defaultMsg: ''};
+    const description = {id: '', defaultMsg: ''};
+
+    if (!isPaidTier) {
+        title.id = 'admin.billing.subscription.privateCloudCard.title';
+        title.defaultMsg = 'Looking for a high-trust private cloud?';
+
+        description.id = 'admin.billing.subscription.privateCloudCard.description';
+        description.defaultMsg = 'If you need software with dedicated, single-tenant architecture, Mattermost Private Cloud (Beta) is the solution for high-trust collaboration.';
+    } else if (isFreeTrial) {
+        title.id = 'admin.billing.subscription.privateCloudCard.freeTrial.title';
+        title.defaultMsg = 'Questions about your trial?';
+
+        description.id = 'admin.billing.subscription.privateCloudCard.freeTrial.description';
+        description.defaultMsg = 'We love to work with our customers and their needs. Contact sales for subscription, billing or trial-specific questions.';
+    } else {
+        title.id = 'admin.billing.subscription.privateCloudCard.cloudProfessional.title';
+        title.defaultMsg = 'Upgrade to Cloud Enterprise';
+
+        description.id = 'admin.billing.subscription.privateCloudCard.cloudProfessional.description';
+        description.defaultMsg = 'Optimize your processes with VPC Peering, a dedicated AWS account and premium support.';
+    }
```

Comment:
The wording to upgrade to Cloud Enterprise is the same for all now, so no need to separate this condition and it will be the message you have in the `else` for all that is not in Free Trial mode